asset-transfer-events/README.md

```diff
@@ -51,37 +58,44 @@ The smart contract implements (in folder `chaincode-xyz`) following functions to
 - DeleteAsset
 - TransferAsset
 
-Note that the asset transfer implemented by the smart contract is a simplified scenario, without ownership validation, meant only to 
+Note that the asset transfer implemented by the smart contract is a simplified scenario, without ownership validation, meant only to
 demonstrate the use of sending and receiving events.
 
 
-## Running the sample
+## Running the samples
 
 Like other samples, we will use the Fabric test network to deploy and run ths sample. Follow these step in order.
 - Create the test network and a channel
-``` 
+```
 cd test-network
 ./network.sh up createChannel -c mychannel -ca
 ```
 
 - Deploy the chaincode (smart contract)
-``` 
+```
 # to deploy javascript version
 ./network.sh deployCC -ccs 1  -ccv 1 -ccep "OR('Org1MSP.peer','Org2MSP.peer')"  -ccl javascript -ccp ./../asset-transfer-events/chaincode-javascript/ -ccn asset-transfer-events-javascript
 
 # or to deploy java version
 ./network.sh deployCC -ccs 1  -ccv 1 -ccep "OR('Org1MSP.peer','Org2MSP.peer')"  -ccl java -ccp ./../asset-transfer-events/chaincode-java/ -ccn asset-transfer-events-java
 ```
 
-- Run the application
+- Run the application javascript
 ```
 cd application-javascript
 npm install
 # ensure this line in app.js have correct chaincode deploy name
 #       const chaincodeName = '...';
 node app.js
 ```
-
+- Run the application gateway typescript
+```
+cd application-gateway-typescript
+npm install
+# ensure this line in app.js have correct chaincode deploy name
+#       const chaincodeName = '...';
```

Comment:
Just running the commands in this README as-is results in a failure running the client application since the chaincode name doesn't match the on used at deployment. I think the sample should work out-of-the-box and not require the user to modify the sample code before it will work.

I would suggest changing the `./network.sh deployCC` command above so specify the same chaincode name as the sample code:

```bash
./network.sh deployCC -ccs 1 -ccv 1 -ccep "OR('Org1MSP.peer','Org2MSP.peer')" -ccl java -ccp ./../asset-transfer-events/chaincode-java/ -ccn events
```